src/Service/CarrierConfigurationProvider.php

```diff
@@ -3,31 +3,60 @@
 namespace Gett\MyparcelBE\Service;
 
 use Db;
-use Validate;
+use Gett\MyparcelBE\Database\Table;
 
 class CarrierConfigurationProvider
 {
+    /**
+     * @var array
+     */
     public static $configuration;
 
+    /**
+     * @var string
+     */
+    protected static $table = Table::TABLE_PRODUCT_CONFIGURATION;
+
+    /**
+     * @param  int    $carrier_id
+     * @param  string $name
+     * @param  null   $default
+     *
+     * @return null|mixed
+     * @throws \PrestaShopDatabaseException
+     */
     public static function get(int $carrier_id, string $name, $default = null)
     {
-        if (!isset(static::$configuration[$carrier_id][$name])) {
-            $result = Db::getInstance()->executeS('SELECT name,value FROM ' . _DB_PREFIX_ . "myparcelbe_carrier_configuration WHERE id_carrier = '{$carrier_id}' ");
+        if (! isset(static::$configuration[$carrier_id][$name])) {
+            $table  = Table::withPrefix(Table::TABLE_CARRIER_CONFIGURATION);
+            $result = Db::getInstance()
+                ->executeS(
+                    <<<SQL
+SELECT name,value FROM `$table` WHERE id_carrier = $carrier_id 
```

Comment:
Sanity Check: de quotes doen we nergens om de `$table` is dat hier ook wenselijk?

Reply:
ik ben niet echt een SQL'er maar het mag en werkt allebei 🤷🏼‍♀️